src/views/received_sms_list/ReceivedSmsList.js

```diff
@@ -7,47 +7,49 @@ import {
     RECEIVED_SMS_LIST_PATH,
     STATUS_ALL,
 } from './config'
-import { DeleteSelectedButton } from './DeleteSelectedButton'
+import DeleteSelectedButton from '../../delete_selected_button/DeleteSelectedButton'
 import { Filter } from './Filter'
 import { PageHeadline } from '../../headline'
 import { SmsTable } from './SmsTable'
 import { dataTest } from '../../dataTest'
-import { useQueryParams } from './useQueryParams'
+import { useQueryParams } from '../../hooks'
 import i18n from '../../locales'
 import styles from './ReceivedSmsList.module.css'
 
-const query = {
-    inboundSms: {
-        resource: 'sms/inbound',
-        params: ({ page, pageSize, phoneNumber, status }) => {
-            const params = {
-                page,
-                pageSize,
-                fields: [
-                    'id',
-                    'text',
-                    'originator',
-                    'smsstatus',
-                    'user[userCredentials[username]]', // sender
-                    'receiveddate',
-                ],
-                order: 'receiveddate:desc',
-            }
+const parseParams = ({ page, pageSize, phoneNumber, status }) => {
+    const params = {
+        page,
+        pageSize,
+        fields: [
+            'id',
+            'text',
+            'originator',
+            'smsstatus',
+            'user[userCredentials[username]]', // sender
+            'receiveddate',
+        ],
+        order: 'receiveddate:desc',
+    }
 
-            const filters = []
-            if (phoneNumber) {
-                filters.push(`originator:ilike:${phoneNumber}`)
-            }
-            if (status && status !== STATUS_ALL) {
-                filters.push(`smsstatus:eq:${status}`)
-            }
+    const filters = []
+    if (phoneNumber) {
+        filters.push(`originator:ilike:${phoneNumber}`)
+    }
+    if (status && status !== STATUS_ALL) {
```

Comment:
I see you've replaced the STATUS_ALL var with its value 'ALL' in other areas. I was wondering why you didn't do that here.

Reply:
I replaced `STATUS_ALL` with `'ALL'` for the shared behaviour like `createSearchString` since `STATUS_ALL` was only defined in the `receved_sms_list` view. I was thinking of delaying more extensive refactoring to another PR but I might as well do it now.

Reply:
Ah yes, that makes sense. All right, whichever you prefer. Leaving it for another PR if it's going to be too much for this one sounds fine to me, if you'd prefer that as well.